spec/factories/service_provider_identities.rb

```diff
@@ -7,4 +7,12 @@
   trait :active do
     last_authenticated_at { Time.zone.now }
   end
+
+  trait :consented do
+    last_consented_at { Time.zone.now - 5.minutes }
+  end
```

Comment:
I actually think we probably want to make `:active` set `last_consented_at`, but maybe _after_ this story.